storage.go

```diff
@@ -494,6 +516,10 @@ func (storage DBStorage) StoreTableIntoFile(tableName TableName,
 func (storage DBStorage) ReadRecordsCount(tableName TableName) (int, error) {
 	sqlStatement := selectCountFromTable(tableName)
 
+	if storage.config.EnableOrgIDFiltering && selectiveExportAllowed(tableName) {
```

Comment:
the same code as on line 352, prob. worth refactoring condition into one method + appending JOIN clause into second method?

Reply:
@tisnik done, TY!